cmd/kube-scheduler/app/server.go

```diff
@@ -482,38 +482,38 @@ func makeLeaderElectionConfig(config componentconfig.KubeSchedulerLeaderElection
 // embed the metrics handler if the healthz and metrics address configurations
 // are the same.
 func makeHealthzServer(config *componentconfig.KubeSchedulerConfiguration) *http.Server {
-	mux := mux.NewPathRecorderMux("kube-scheduler")
-	healthz.InstallHandler(mux)
+	mutex := mux.NewPathRecorderMux("kube-scheduler")
```

Comment:
If you'd like to rename the variable, you'd better not use `mutex` as its name  which might cause confusing. A `Mux` is not a `Mutex` in the context.

FYI:
https://github.com/gorilla/mux

> The name mux stands for "HTTP request multiplexer".